digitalstrategie/settings/base.py

```diff
@@ -173,6 +174,12 @@
 # Wagtail settings
 WAGTAIL_SITE_NAME = "digitalstrategie"
 WAGTAILIMAGES_IMAGE_MODEL = 'apps_images.CustomImage'
+WAGTAILEMBEDS_FINDERS = [
+    {
+        'class': 'wagtail.embeds.finders.oembed',
+        'providers': [youtube, vimeo],
+    }
+]
```

Comment:
not sure if this is the best place to add this?!

Reply:
Yes, I think it does fit with the images